radixdlt/src/main/java/com/radixdlt/middleware2/network/GetVerticesResponseMessage.java

```diff
@@ -31,23 +33,33 @@
  */
 @SerializerId2("message.consensus.vertices_response")
 public final class GetVerticesResponseMessage extends Message {
+	@JsonProperty("vertexId")
+	@DsonOutput(Output.ALL)
+	private final Hash vertexId;
+
 	@JsonProperty("vertices")
 	@DsonOutput(Output.ALL)
 	private final List<Vertex> vertices;
 
 	GetVerticesResponseMessage() {
 		// Serializer only
 		super(0);
+		this.vertexId = null;
 		this.vertices = null;
 	}
 
-	GetVerticesResponseMessage(int magic, List<Vertex> vertices) {
+	GetVerticesResponseMessage(int magic, Hash vertexId, List<Vertex> vertices) {
 		super(magic);
+		this.vertexId = Objects.requireNonNull(vertexId);
 		this.vertices = Objects.requireNonNull(vertices);
 	}
 
+	public Hash getVertexId() {
+		return vertexId;
+	}
+
 	public List<Vertex> getVertices() {
-		return vertices;
+		return vertices == null ? Collections.emptyList() : vertices;
```

Comment:
Would prefer this to be returning an immutable list, so that this object remained immutable.